libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/defaults.parm

```diff
@@ -1,11 +1,17 @@
 # setup correct defaults for battery monitoring for cube power brick
 BATT2_CURR_PIN 4
 BATT2_VOLT_PIN 13
+BATT_MONITOR 4
 BATT_AMP_PERVLT 39.877
 BATT_VOLT_MULT 12.02
 BATT2_AMP_PERVLT 39.877
 BATT2_VOLT_MULT 12.02
+
 # setup ADSB
 ADSB_TYPE 1
 SERIAL5_BAUD 57
 SERIAL5_PROTOCOL 1
+CAN_P1_DRIVER 1
+CAN_P2_DRIVER 1
+NTF_LED_TYPES 231
+GPS_TYPE 9
```

Comment:
this will break a lot of existing setups that use serial GPS (eg. F9P) on CubeOrange